components/BB.js

```diff
@@ -1,85 +1,33 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Grid } from "@material-ui/core";
-import AssignmentTurnedIn from "./icons/AssignmentTurnedIn";
-import SaveChecked from "./icons/SaveChecked";
 import { connect } from "react-redux";
-import {
-  getFavouritesUrl,
-  getPrintUrl,
-  getHomeUrl,
-  getSummaryUrl
-} from "../selectors/urls";
+import { getHomeUrl } from "../selectors/urls";
 import { css } from "emotion";
 import Container from "../components/container";
-import HeaderButton from "./header_button";
-import HeaderLink from "./header_link";
 import { globalTheme } from "../theme";
 import { DisabledCookiesBanner } from "./disabled_cookies_banner";
 import { areCookiesDisabled } from "../utils/common";
 import BenefitsPane from "./benefits_pane";
 import BreadCrumbs from "../components/breadcrumbs";
-import ShareBox from "../components/share_box";
-import EditIcon from "./icons/Edit";
 import Cookies from "universal-cookie";
 import Paper from "./paper";
+import Header from "./typography/header";
+import NextSteps from "./next_steps";
+import QuickLinks from "./quick_links";
+import StickyHeader from "./sticky_header";
 
+const divider = css`
+  border-top: 2px solid ${globalTheme.colour.duckEggBlue};
+  width: 100%;
+`;
 const outerDiv = css`
   padding-bottom: 16px !important;
 `;
 const topMatter = css`
   background-color: ${globalTheme.colour.white};
   width: 100%;
 `;
-const sidebarLinks = css`
-  // for big screen...
-  @media only screen and (min-width: ${globalTheme.min.xs}) {
-    padding: 1em 24px !important;
-    border-top: thin solid ${globalTheme.colour.paleGreyishBrown};
-    border-bottom: thin solid ${globalTheme.colour.paleGreyishBrown};
-  }
-  margin-bottom: 24px;
-`;
-const sidebar = css`
-  position: -webkit-sticky;
-  position: sticky;
-  top: 0;
-  background-color: ${globalTheme.colour.white};
-  z-index: 10;
-`;
-
-// if screen size is max.xs or smaller, hide long text
-const longText = css`
-  @media only screen and (max-width: ${globalTheme.max.xs}) {
-    display: none !important;
-  }
-`;
-// if screen size is min.xs or larger, hide short text
-const shortText = css`
-  @media only screen and (min-width: ${globalTheme.min.xs}) {
-    display: none !important;
-  }
-`;
-const savedListLink = css`
-  @media only screen and (max-width: ${globalTheme.max.xs}) {
-  }
-`;
-const editLink = css`
-  @media only screen and (max-width: ${globalTheme.max.xs}) {
-  }
-`;
-const hideOnMobile = css`
-  // if screen size is max.xs or smaller
-  @media only screen and (max-width: ${globalTheme.max.xs}) {
-    display: none !important;
-  }
-`;
-const showOnMobile = css`
```

Comment:
Some of this styling moved to `StickyHeader` and some of the mobile specific styling is now unnecessary since the styling is now the same on mobile and desktop.